contents/newsletter/first-time-founders.md

```diff
@@ -0,0 +1,126 @@
+
+3. **Hiring people you don't need:** Paul recommends a "need-based" approach to hiring. "Only hire someone when you have a problem that's big enough for a full-time hire." Not a theoretical problem or an opportunity, a burning problem you can only solve by hiring someone. "We now have around 80 people at Supabase, so I usually say that we have had 80 problems worth solving. This forces you to do less, stay focused, and maintain velocity."
+
+## Taylor Wakefield (Co-founder of Mailgun and Teleport)
+
+- **Profista:** 2008 to 2010 – dashboards and collaboration tools for general managers
+- **Mailgun:** 2010 to 2012 – acquired by Rackspace, now has 150k+ customers
+- **Teleport:** 2015 to present – used by dbt Labs, DoorDash and Nasdaq
+
+THIS WILL BE A GRAPHIC
+
+Before [Taylor Wakefield](https://www.linkedin.com/in/taylorwakefield/ found success with Mailgun and Teleport, he co-founded Profista, a web app and consulting company that helped business leaders rack financial results and "operational drivers." 
+
+> "We essentially sold BI vaporware to large companies like EMC and Cisco to help manage their business. We would use Tableau graphs embedded in wireframes to demonstrate the idea to them."
+
+They generated $500k of orders this way and worked closely with management to build the workflows. Users would email spreadsheets to an endpoint, which would then automagically populate graphs in the app. 
+
+Early feedback was glowing and the product "blew the customers away," says Taylor. There was just one teeny, tiny problem. No one used it.
+
+> "We didn't do a good job of working with the individuals who were going to be actually using the product on a day-to-day basis. It was pretty clear that it didn't make their lives easier, and they were only using it because their boss told them to. In the end, the actual users revolted, and the product didn't give the managers enough value for them to keep making their employees use it."
+
+They persevered for a while in the hope of turning things around, but it never worked out.
+
+### What Taylor learned
+
+Initial interest and orders convinced Taylor they were onto something, but they made a few classic first-time mistakes:
+
+1. **Working on a problem they didn't know well:** While Taylor and his co-founders knew there was a problem to solve, it wasn't one they had personally encountered or tried to solve. In contrast, when Taylor and Ev Kontsevoy founded Mailgun, "Ev was solving his own problem that he understood deeply." They quickly found product-market fit and Mailgun was acquired after just two years.
+
+2. **Talking to the wrong people:** Managers have opinions, but they rarely understand the challenges of their employees. By working with managers as design partners, rather than the people who would use the app, Profista created a solution that didn't survive contact with reality.
```

Comment:
```suggestion
2. **Talking to the wrong people:** Managers have opinions, but rarely understand the challenges of their employees. By working with managers as design partners, rather than the people who would use the app, Profista created a solution that didn't survive contact with reality.
```